setup.py

```diff
@@ -7,7 +7,7 @@
 import sys
 from setuptools import setup, find_packages
 
-LIBRARY_VERSION = '1.0.9'
+LIBRARY_VERSION = '1.1.0'
```

Comment:
Bumping the minor version for prescriptor work phase 2
Phase 1 = predictor = 1.0.* versions
Phase 2 = prescriptor = 1.1.* versions